pyproject.toml

```diff
@@ -81,6 +81,9 @@ uwsgitop = "^0.12"
 pytest-lazy-fixtures = "^1.1.1"
 pycountry = "^24.6.1"
 qdrant-client = {extras = ["fastembed"], version = "^1.12.0"}
+onnxruntime = "1.19.2"
+openai = "^1.55.3"
+litellm = "^1.53.5"
```

Comment:
Is it important to have this exact onnxruntime version?  `1.20.0` is already included in the main branch.

If not I think you can remove this, and openai too, just add `litellm` - litellm will bring in openai anyway (version 1.57.0)

I tried it out by resetting both pyproject.toml and poetry.lock to the `main` branch versions and adding only litellm, then regenerated embeddings.  Everything still seemed work.

Reply:
I had hit merge just before catching this. Yes there was a reason for that specific pin. I can't find the exact thread but using the latest onnxruntime had issues with certain models. I was seeing it locally where the process consistently got killed and the only way around it was to downgrade (similarly suggested here https://github.com/microsoft/onnxruntime/issues/10039) -  my suggestion is to leave atleast the onnyxruntime pin for now unless there is a compelling reason to upgrade. ill see if I can put together a repro case